.github/workflows/build_mpv_binary_and_deb_package.yml

```diff
@@ -0,0 +1,139 @@
+name: Build mpv binary and create debian package
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+            build-essential \
+            clang \
+            yasm \
+            git \
+            meson \
+            ninja-build \
+            autoconf \
+            automake \
+            libtool \
+            pkg-config \
+            libfreetype6-dev \
+            libfribidi-dev \
+            libfontconfig1-dev \
+            libasound2-dev \
+            pulseaudio \
+            libx11-dev \
+            libxss-dev \
+            libvdpau-dev \
+            libgl-dev \
+            libxv-dev \
+            libjpeg-dev \
+            libssl-dev \
+            libx264-dev \
+            libmp3lame-dev \
+            libfdk-aac-dev \
+            python3-pip \
+            python3.12-venv \
```

Comment:
do we use venv or pip anywhere?

Reply:
No actually. I used these lines to download `youtube-dl` by using python, because the package was not available in the ubuntu repositories but it is only a run time dependency. I guess we can delete these packages safely. Thank you.